akka-actor/src/main/scala/akka/dispatch/Dispatchers.scala

```diff
@@ -77,7 +90,12 @@ class Dispatchers(val settings: ActorSystem.Settings, val prerequisites: Dispatc
 
```

Comment:
we will also need `private[akka] def internalDispatcher: MessageDispatcher` here